bin/shorebird.ps1

```diff
@@ -132,7 +132,7 @@ function Update-Shorebird {
     Update-Flutter
 
     Push-Location $shorebirdCliDir
-    & $dart pub upgrade
+    & $dart pub update
```

Comment:
I wasn't able to test this change since I don't have a windows machine. This shouldn't fail since I just changes `upgrade` to `update`, but I would appreciate if someone could help me test this just to be sure

Reply:
Isn't it "get"?

Reply:
See, that is why we do code review 🤦

Reply:
Update and upgrade have the same behavior afaict

Reply:
yeah, it should be `pub get` instead, I just have too many open threads in my head right now and confused things up hahaha

Gonna change this!